core/src/main/java/org/apache/carbondata/core/statusmanager/SegmentUpdateStatusManager.java

```diff
@@ -80,15 +68,25 @@
 
   public SegmentUpdateStatusManager(CarbonTable table,
       LoadMetadataDetails[] segmentDetails) {
+    this(table, segmentDetails, null);
+  }
+
+  public SegmentUpdateStatusManager(CarbonTable table,
```

Comment:
please add comment to describe the content of updateVersion, give an example of updateVersion

Reply:
ok